Makefile

```diff
@@ -115,9 +115,7 @@ generate-grpc:  ## generate grpc code
 	 	--pyi_out=. \
 	 	proto/action_webhook.proto
 
-check-generate-grpc-code-in-sync: generate-grpc
-check-generate-grpc-code-in-sync: ## check if the generated code is in sync with the proto files
-	# this is a helper to check if the generated code is in sync with the proto files
+check-generate-grpc-code-in-sync: generate-grpc ## check if the generated code is in sync with the proto files, it uses a helper to check if the generated code is in sync with the proto files
```

Comment:
Have you validated if this is not breaking `make help` target? Comment should be displayed as description for the target.

Reply:
@radovanZRasa Yes 

```
❯ make help
...
check-generate-grpc-code-in-sync check if the generated code is in sync with the proto files, it uses a helper to check if the generated code is in sync with the proto files
...
```